site/components/map/ServicesMap.tsx

```diff
@@ -222,120 +224,129 @@ const Map = ({
         setPopupInfo({});
     }, []);
 
-    const selectAllStops = async (evt: SyntheticEvent) => {
-        evt.preventDefault();
+    const addSelectedStopsAndServices = async (includeMarkerData?: boolean) => {
```

Comment:
If I use the polygon tool first and then I select a service, the polygon tool remains. Is this the intended functionality or should the polygon tool be cleared?

See screen recording below:
https://github.com/Department-for-Transport-Disruptions/create-disruptions-data/assets/83244002/1d0714bf-650e-4c0d-8700-130450357814

Reply:
intended it just shouldn't select stops from the polygon and it should select those in that service